server/controllers/content.js

```diff
@@ -414,10 +416,36 @@ export function renderNewsletterPage(ctx, next) {
 export async function searchForDrupalRedirect(ctx, next) {
   const {path} = ctx.params;
   const page = await getPageFromDrupalPath(ctx.request, `/${path}`);
+  const baseGaPayload = {
+    v: 1,
```

Comment:
What is `v` and `cid`

Reply:
`v` is [protocol version](https://developers.google.com/analytics/devguides/collection/protocol/v1/parameters#v) (currently version 1).

`cid` is [client ID](https://developers.google.com/analytics/devguides/collection/protocol/v1/parameters#cid) – used to 'anonymously identify a particular user, device, or browser instance, stored as a first-party cookie with a two-year expiration'. It's required, but don't know if we want to generate/read these, or just leave it hard-coded.

Reply:
`cid` needs to UUID V4 compliant.
https://www.npmjs.com/package/uuid